README.md

```diff
@@ -52,27 +69,41 @@ This will run your jasmine tests, loading all the tests with JSPM instead of nod
 ##### Options:
 ```js
 runTests({
-    // Provide a custom path to jasmine.json config file.
-    // Defaults to path where the script is being
-    // executed + '/spec/support/jasmine.json'.
-    jasmineConfig: "PATH/TO/JASMINE/JSON/RELATIVE/TO/CWD/config.json",
-
-    // Provide a custom path to package.json config file.
-    // Defaults to path where the script is being executed.
-    packagePath: "PATH/TO/PACKAGE/JSON/RELATIVE/TO/CWD/"
-})
-```
 
-Alternatively, you can pass the jasmine config object directly.
-```js
-runTests({
-    // Provide a custom jasmine configuration without a jasmine.json.
-    jasmineConfig: {
-        "spec_dir": "src/test/specs",
-        "spec_files": ["**/*.js"],
-        "helpers": []
-    },
-    packagePath: "PATH/TO/PACKAGE/JSON/RELATIVE/TO/CWD/"
+  // Provide a custom jasmine.json config file
+  // Defaults to path where `jspmjasmine` is being executed + `/spec/support/jasmine.json
+  jasmineConfig: "RELATIVE/PATH/TO/jasmine.json",
+  // Alternatively, you can pass the jasmine config object directly.
+  // jasmineConfig: {
+    // "spec_dir": "src/test/specs",
+    // "spec_files": ["**/*.js"],
+    // "helpers": []
+  // },
+
+  // Provide a custom directory to package.json
+  // Defaults to path where `jspmjasmine` is being executed
+  packagePath: "RELATIVE/PATH/TO/PACKAGE/JSON/DIRECTORY/",
+
+  // Add a coverage report
+  // Defaults to undefined
+  coverage: {
+
+    // Provide a custom directory for storing coverage reports
+    // Defaults to "./coverage/"
+    dir: "RELATIVE/PATH/TO/COVERAGE/DIRECTORY/",
+
+    // Provide a custom type for coverage reports
+    // Defaults to "html" (all report types shipped with Istanbul are supported)
+    reporter: "TYPE",
+
+    // Provide files to be instrumented by Istanbul coverage
+    // Defaults empty array (no files)
+    files: "RELATIVE/GLOB/PATH/TO/FILES/**/*.js"
+  },
+
+  // Allow `node-jspm-jasmine` to empty coverage directory before creating a new report
+  // Defaults to undefined (no clean up)
+  cleanDir: true
```

Comment:
This should be inside of the coverage object. https://github.com/CanopyTax/node-jspm-jasmine/blob/master/src/node-jspm-jasmine.js#L56